server/cluster-setup/clusterSetup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Install Helm
+sudo snap install helm --classic
+
+# Add helm repos 
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx # ngnix ingress controller
+helm repo add jetstack https://charts.jetstack.io # cert manager
+helm repo add bitnami https://charts.bitnami.com/bitnami # redis
+helm repo update
+
+# install packages onto Kubernetes
+helm install sensor ingress-nginx/ingress-nginx --set controller.publishService.enabled=true # install nginx ingress controler
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.6.0 --set installCRDs=true # install cert-manager
+kubectl create namespace redis
+kubectl apply -f redis-storage.yaml
+helm install sensor-redis bitnami/redis --namespace redis --set persistence.storageClass=redis-storage --set auth.enabled=false --set architecture=standalone
```

Comment:
What do you think about specifying the version of these helm charts to avoid problems of flags changing? It looks like it can be done with the `--version` flag like the `cert-manager` on line 14.

Reply:
let me look into this, because that flag I think refers to the cert-manager version, not the chart version - and you can have more than one chart (and accompanying flags) for the same app version. Will update after checking

Reply:
Thanks :smile:

Reply:
We didn't end up doing this.